content/blog/page-view-counter/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Using Azure Functions to create a page view counter
+date: "2019-12-26T22:12:03.284Z"
+author: nacho
+description:
+  "We can extend static sites by creating a simple page view counter that is processed serverless, and for free. Let's try to build it on Microsoft Azure services."
+tags:
+  - azure functions
+  - storage
+  - .NET
+  - cloud
+---
+
+## Introduction
+
+Since we are using [Gatsby](https://www.gatsbyjs.org/) to build this site as a set of static files, there are some common functionalities that you might find in dynamic sites which you will naturally be missing. However, this does not mean they cannot be implemented in some way.
+
+As a way of testing the development of a small, simple feature and some Azure offerings, we decided to try to build a page view counter for our blog posts.
+
+## Scope
+
+The scope of this small experiment will be to build a small service that maintains the state of how many times a specific blog post (or in general, a page) was viewed. Since in our case we are building the site statically, we will be calling the service from the frontend.
+
+## Tech requirements
+
+ Azure provides [multiple compute solutions](https://docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-overview), most of which will likely not be optimal in certain use cases. Considering the scope, [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview) sounds like a good offering. Function apps are serverless execution contexts where we can run a piece of code (the Functions themselves) in a stateless way. 
```

Comment:
`serverless` is a super buss-word and should be used in `tags` and in `descriptions`

Reply:
Sorry, I only know of "buzzwords", not "busswords". Jk, good tip.

Reply:
It was on the description already BTW, and I wanted to add it in the tags but didn't want the section to become too bloated. Still, I readded it.